extensions-core/parquet-extensions/pom.xml

```diff
@@ -411,11 +403,6 @@
       <artifactId>api-util</artifactId>
       <scope>provided</scope>
     </dependency>
-    <dependency>
-      <groupId>org.apache.avro</groupId>
-      <artifactId>avro</artifactId>
-      <scope>provided</scope>
```

Comment:
How about keeping this in, but without `<scope>provided</scope>`.

It seems like the dependency really is needed, but it is not provided, so it should have normal scope.

Reply:
makes sense to me. Done